### ISSUE TITLE
Navigate directly to gist file fragment

### ISSUE THREAD

**Issue by Bluscream** [slapperwan/gh4a#828]
When clicking on a url like https://gist.github.com/Bluscream/119f09441c512ef267ade38bd4a5c9ce#file-ts_bad_hotkeys-ahk it could know which file i wanted to see :)

**Comment by maniac103** [comment]
Yes, but there's one caveat: I don't know the exact file name transformation Github uses. In your example, `ts_bad_hotkeys.ahk` is transformed to `ts_bad_hotkeys-ahk`.
I can replace the dot, but there likely are more characters that are replaced as well.

**Comment by Bluscream** [comment]
url_encode maybe?
